### PR TITLE
feat: add production ready compose file

### PR DESCRIPTION
**Pull request by c3ppc3pp**
This Pull request adds a compose file which can be used in production.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+      - SMTP_URI=${SMTP_URI}
+      # If you don't have a mailserver you can use the provided mailslurper
+      # but keep in mind, this one is insecure, only use it at your own risk!
+      # - SMTP_URI=smtps://test:test@mailslurper:1025/?skip_ssl_verify=true&legacy_ssl=true
+    depends_on:
+      crux-postgres:
+        condition: service_started
+      crux-migrate:
+        condition: service_completed_successfully
+      proxy:
```
Comment (polaroi8d) [inline docker-compose.yaml:49]: service "crux-ui" depends on undefined service proxy: invalid compose project

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# Cookie/JWT secrets
+CRUX_SECRET=Random_Generated_String
+KRATOS_SECRET=Random_Generated_String
+
+# Mailserver settings
+SMTP_URI=smtps://username:password@mailserver.example.com:465
+
+# Recaptcha secrets
+RECAPTCHA_SECRET_KEY=Recaptcha_Secret_Key
+RECAPTCHA_SITE_KEY=Recaptcha_Site_Key
```
Comment (polaroi8d) [inline .env.example:19]: Minor: No NL

**Review by polaroi8d — CHANGES_REQUESTED** [review]
Tested, not working.

File: docker-compose.yaml (hunk 1 above)
Comment (c3ppc3pp) [reply docker-compose.yaml]: It was renamed on the test-env after running into some issues with container naming. Thanks, fixed.

**Review by polaroi8d — APPROVED** [review]
Tested, works.

**Pull request merged**